js/views/userPageVw.js

```diff
@@ -389,13 +391,10 @@ module.exports = Backbone.View.extend({
           self.model.set('page', tempPage);
           self.$el.find('.js-userPageBanner').css('background-image', 'url(' + server + "get_image?hash=" + imageHash + ')');
         }else if (data.success === false){
-          var errorModal = $('.js-messageModal');
-          errorModal.removeClass('hide');
+          //TODO: Extract these lines to a more generic showModal-type function
           errorModal.find('.js-messageModal-title').text("Changes Could Not Be Saved");
           errorModal.find('.js-messageModal-message').html("Uploading the image has failed due to the following error: <br/><br/><i>" + data.reason + "</i>");
-        }else if (imageHash == "b472a266d0bd89c13706a4132ccfb16f7c3b9fcb") {
-          var errorModal = $('.js-messageModal');
-          errorModal.removeClass('hide');
+        }else if (imageHash == "b472a266d0bd89c13706a4132ccfb16f7c3b9fcb") { //TODO: Remove magic number
```

Comment:
What? You're suggesting leaving in a raw reference to b472a266d0bd89c13706a4132ccfb16f7c3b9fcb?